src/gtc/utils.py

```diff
@@ -38,3 +38,9 @@ def flatten_list_iter(
         else:
             if item is not None or not filter_none:
                 yield item
+
+
+def mask_to_dims(mask: Tuple[bool, bool, bool]) -> str:
```

Comment:
The naming is somewhat strange, you pass something you called `dimensions` in the `FieldDecl`, but call it `mask` here and get something you call `dims`. Maybe:

```suggestion
def dimension_mask_to_str(mask: Tuple[bool, bool, bool]) -> str:
```

Reply:
has been changed